### PR TITLE
Repositorys

### PR DESCRIPTION
**Pull request by mrLag0fan**

### CODE REVIEW

File: Domain/FolderEntity.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Domain;
+
+public class FolderEntity
+{
+    public int Id { get; set; }
```
Comment (mirkru37) [inline Domain/FolderEntity.cs:5]: Please make entity with id and inherit other from it. ID should be[ UUID ](https://stackoverflow.com/questions/23081096/entity-framework-6-guid-as-primary-key-cannot-insert-the-value-null-into-column)

File: Domain/TagEntity.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Domain;
+
+public class TagEntity
+{
+    public int Id { get; set; }
```
Comment (mirkru37) [inline Domain/TagEntity.cs:5]: Same

File: Domain/FolderEntity.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    public string Name { get; set; }
+    public bool IsPublic { get; set; }
+    public virtual List<TaskEntity> Tasks { get; set; }
+    public virtual List<UserEntity> Users { get; set; }
```
Comment (mirkru37) [inline Domain/FolderEntity.cs:9]: Why it is list? Folder could be owned by multiple users? My suggest is to add owner field and contributors

File: Domain/TaskEntity.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+﻿namespace Domain;
+
+public class TaskEntity
```
Comment (mirkru37) [inline Domain/TaskEntity.cs:3]: Please add cteatedAt and deletedAt fields.

File: Domain/TaskEntity.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+{
+    public int Id { get; set; }
```
Comment (mirkru37) [inline Domain/TaskEntity.cs:5]: Same with id

File: WebUI/Views/Shared/_TestPartial.cshtml [hunk 6]
```diff
@@ -1,2 +1,2 @@
-<b>Partial 1</b>
+<b>${}</b>
```
Comment (mirkru37) [inline WebUI/Views/Shared/_TestPartial.cshtml:1]: please remove this change

File: identifier.sqlite
Comment (mirkru37) [inline identifier.sqlite:1]: What is this file?

File: WebUI/Pages/User/Index.cshtml
Comment (mirkru37) [inline WebUI/Pages/User/Index.cshtml:1]: Why we need this?

**Review by mirkru37 — CHANGES_REQUESTED** [review]

File: Domain/FolderEntity.cs (hunk 3 above)
Comment (mrLag0fan) [reply Domain/FolderEntity.cs:9]: це всі юзери які мають доступ до тої папки

File: WebUI/Pages/User/Index.cshtml
Comment (mrLag0fan) [reply WebUI/Pages/User/Index.cshtml:1]: подумай

File: identifier.sqlite
Comment (mirkru37) [reply identifier.sqlite:1]: We shouldn't pass database files to our pero, please add this to .gitignore

File: WebUI/Pages/User/Index.cshtml
Comment (mirkru37) [reply WebUI/Pages/User/Index.cshtml:1]: We don't need User controller as well as views for now, please remove

**Pull request merged**